cpp/modmesh/view/wrap_view.cpp

```diff
@@ -35,6 +35,103 @@
 #include <QPointer>
 #include <QClipboard>
 
+#ifdef MODMESH_USE_PYSIDE
+
+// Usually MODMESH_PYSIDE6_FULL is not defined unless for debugging.
```

Comment:
It's quite messy to directly use PySide/Shiboken headers, so I use the macro to fence the header inclusion.